version.md

```diff
@@ -0,0 +1,78 @@
+# Versioning endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents a single version, matching a single item.
+
+Whether a version is accessible depends on `versioning.item.history.view.admin` configuration.  
+If this is set to true, the version can only be retrieved if the user is an admin of the related item
+
+## Create version
+
+TODO
+
+## Get single version
+
+**GET /api/versioning/versions/<:versionId>**
+
+Provide version information for the version id.
+
+```json
+{
+  "id": "102",
+  "version": "2",
+  "type": "version",
+  "created": "2015-11-03T09:44:46.617",
+  "summary": "Fixing some typos in the abstract",
+  "_links": {
+    "versionhistory": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/10"
```

Comment:
please fix the plural also in the examples

Reply:
This has been updated